packages/trader/src/App/Components/Form/TimePicker/__tests__/dialog.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Dialog from '../dialog';
+import moment from 'moment';
+
+describe('<Dialog />', () => {
+    const startTimes = [moment('2023-11-21T08:00:00')];
+    const endTimes = [moment('2023-11-22T12:00:00')];
+    const onChangeMock = jest.fn();
+    const className = 'testClassName';
+    const preClass = 'testPreClass';
+    const selectedTime = '23:45';
```

Comment:
@henry-deriv  please use snake_case for all variables you created in this file as per our naming convention :)